src/js/vre/chapter36/config/form.js

```diff
@@ -88,30 +105,75 @@ const formConfig = {
               }
             }
           }
-        }
-      }
-    },
-    veteranInformation: {
-      title: 'Veteran Information',
-      pages: {
-      }
-    },
-    additionalInformation: {
-      title: 'Additional Information',
-      pages: {
-      }
-    },
-    militaryHistory: {
-      title: 'Military History',
-      pages: {
-      }
-    },
-    contactInformation: {
-      title: 'Contact Information',
-      pages: {
+        },
+        nonVeteranInformation: {
+          title: 'Applicant information',
+          path: 'non-veteran-information',
+          depends: {
+            'view:isVeteran': false
+          },
```

Comment:
Ticket says this is for non veterans only. Not sure what I should call this page since it wasn't in the list. @jbalboni suggestions?

Reply:
I can't really think of a better name, so this seems alright to me. We can have @stacyhlinn check it when she has time.